scripts/linters/ts_ignore_exceptions.json

```diff
@@ -0,0 +1,145 @@
+{
+  "core/templates/domain/improvements/NeedsGuidingResponsesTaskObjectFactorySpec.ts": [
+    "task_type: '???',"
+  ],
+  "extensions/interactions/LogicProof/static/js/tools/logic-demo-test.controller.spec.ts": [
+    "properties than entries according with lint settings.",
+    "properties than {} according with lint settings."
+  ],
+  "extensions/interactions/CodeRepl/directives/code-repl-validation.service.spec.ts": [
+    "customizationArguments.language.value = 1;",
+    "customizationArguments.placeholder.value = 1;",
+    "customizationArguments.preCode.value = 1;",
+    "customizationArguments.postCode.value = 1;"
+  ],
+  "core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.controller.spec.ts": [
+    "spyOn(window, '$').withArgs('.oppia-profile-image-uploader').and",
+    ".returnValue(element);",
+    "properties than toDataURL according with lint settings."
+  ],
+  "core/templates/pages/topic-editor-page/modal-templates/questions-list-select-skill-and-difficulty-modal.controller.spec.ts": [
+    "expect(summary.isSelected).toBe(true);",
+    "expect(summary.isSelected).toBe(false);"
+  ],
+  "core/templates/mathjaxConfig.ts": ["skipStartupTypeset: true,"],
+  "core/templates/domain/collection/collection-validation.service.ts": [
+    "rid of the TS2339 error on AppConstants."
+  ],
+  "core/templates/services/request-interceptor.service.ts": [
+    "request.body = body;",
+    "request.body = {"
+  ],
+  "extensions/objects/templates/svg-filename-editor.component.spec.ts": [
+    "be declared."
+  ],
+  "core/templates/Polyfills.ts": [
+    "navigator.mediaDevices = {};",
+    "property does not exist error."
+  ],
+  "core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts": [
+    "should be declared.",
+    "1 argument in the lints.",
+    "just offset in the lint settings."
+  ],
+  "core/templates/domain/story/StoryObjectFactorySpec.ts": [
+    "sampleStoryBackendDict);"
+  ],
+  "core/templates/domain/improvements/HighBounceRateTaskObjectFactorySpec.ts": [
+    "task_type: '???',"
+  ],
+  "core/templates/services/navigation.service.spec.ts": [
+    "angularElementSpy = spyOn(angular, 'element').and.returnValue(element);"
+  ],
+  "core/templates/services/assets-backend-api.service.spec.ts": [
+    "be declarated."
+  ],
+  "core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service.spec.ts": [
+    "since sort is an ENUM, we can't assign any random value."
+  ],
+  "core/templates/domain/statistics/PlaythroughIssueObjectFactorySpec.ts": [
+    "piof.createFromBackendDict(playthroughDict);"
+  ],
+  "core/templates/services/schema-default-value.service.ts": [
+    "this.logger.error('Invalid schema type: ' + schema.type);"
```

Comment:
Don't understand the comment above this one. It doesn't explain why the suppression is needed.

Reply:
added explanation